src/controllers/users.js

```diff
@@ -1,16 +1,13 @@
-const { User } = require('../models');
+const { UsersService } = require('../services');
```

Comment:
Where these "users" changes came from? Asen is working on users flow, so remove them